app/main.py

```diff
@@ -1,5 +1,68 @@
+from typing import List
+
+
 class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self, name: str,
+            weight: int,
+            coords: List[int] = None
+    ) -> None:
+        self.coords = [0, 0] if coords is None else coords
+        self.name = name
+        self.weight = weight
+
+    def go_forward(self, step: int = 1) -> None:
+        self.coords[1] += step
+
+    def go_back(self, step: int = 1) -> None:
+        self.coords[1] -= step
+
+    def go_right(self, step: int = 1) -> None:
+        self.coords[0] += step
+
+    def go_left(self, step: int = 1) -> None:
+        self.coords[0] -= step
+
+    def get_info(self) -> str:
+        return f"Robot: {self.name}, Weight: {self.weight}"
+
+
+class FlyingRobot(BaseRobot):
+    def __init__(
+            self, name: str,
+            weight: int,
+            coords: List[int] = None
+    ) -> None:
+        coords = [0, 0, 0] if coords is None else coords
+        super().__init__(name, weight, coords)
```

Comment:
There is no need to define `coords` variabe, you could pass value in the `__init__` directly
```suggestion
        super().__init__(name, weight, coords or [0, 0, 0])
```